components/Schedule/ScheduleItemMap.jsx

```diff
@@ -0,0 +1,89 @@
+export const ScheduleItemMap = new Map([
```

Comment:
y we usin .jsx instead of tsx? 

and what is this Map function, y don't u just use do scheduleitemMap = {...}